ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/ProcessingLogConfig.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.util;
+
+import io.confluent.ksql.util.KsqlConfig;
+import java.util.Map;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.apache.kafka.common.config.ConfigDef.ValidString;
+import org.apache.kafka.common.config.ConfigDef.Validator;
+
+public class ProcessingLogConfig extends AbstractConfig {
+  private static final String PROPERTY_PREFIX = "processing.log.";
+
+  private static String propertyName(final String name) {
+    return KsqlConfig.KSQL_CONFIG_PROPERTY_PREFIX + PROPERTY_PREFIX + name;
+  }
+
+  public static final String AUTO_CREATE_ON = "on";
+  public static final String AUTO_CREATE_OFF = "off";
+
+  public static final String STREAM_NAME = propertyName("stream.name");
+  private static final String STREAM_NAME_DEFAULT = "KSQL_PROCESSING_LOG";
```

Comment:
Do we include the service id in the default name to avoid topic clash if two clusters are run against the same kafka cluster?